tests/integration/modules/test_esxi.py

```diff
@@ -124,6 +124,20 @@ def test_esxi_host_capability_params(service_instance, integration_test_config,
         assert capabilities[host_id][arg_name] == expected_value
 
 
+def test_list_pkgs(service_instance):
```

Comment:
The test seems a little weak - we verify that nothing raises and that there is some data within host?

I'm assuming that at least one package is always installed on a host by default, how about we verify that it contains data for the 7 keys we load?